### PR TITLE
Windows start animation

### PR DESCRIPTION
**Pull request by SaumyaKumar-09**
**Just like windows start but a bit different**

### CODE REVIEW

File: include.js [hunk 1]
```diff
@@ -27,20 +34,6 @@ let cards = [
...
-    author: 'SaumyaKumar-09',
-    githubLink: 'https://github.com/SaumyaKumar-09'
-  },
-  {
-    artName: 'Windows start animation',
-    pageLink: './Art/SaumyaKumar-09/Animation%202/index.html',
-    imageLink: './Art/SaumyaKumar-09/Animation%202/Animation2.gif',
-    author: 'SaumyaKumar-09',
-    githubLink: 'https://github.com/SaumyaKumar-09'
-  },
```
Comment (LaurelineP) [inline include.js:43]: # Feedback -  wrong folder naming
- please remove any space in folder naming ( all animation folder ) and use hyphens or underscore instead

Comment (SaumyaKumar-09) [reply include.js:61]: Okay. I changed as you requested.

**Comment by SaumyaKumar-09** [comment]
@sophiabrandt I've committed the required changes. So please review this again and let me know .

**Comment by sophiabrandt** [comment]
@SaumyaKumar-09 There are still empty spaces in your file and folder names, see comment by LaurelineP.

**Pull request closed without merging**